lang/rust-bootstrap/files/patch-no_wrong_assembly.diff

```diff
@@ -8,7 +8,7 @@ Use of muniversal PG ensures CMAKE_OSX_ARCHITECTURES is only one value,
      endif()
    else()
      check_symbol_exists(__x86_64__ "" IS_X64)
--    if (IS_X64 OR CMAKE_OSX_ARCHITECTURES)
+-    if (IS_X64 OR CMAKE_OSX_ARCHITECTURES MATCHES "x86_64")
```

Comment:
I *suspect* that this diff is no longer necessary because it looks explicitly for x86_64 instead of just assuming the way it previously did. We’d need to look at the `git blame` of the target file.